org-formation/050-costs/_tasks.yaml

```diff
@@ -11,3 +11,14 @@ AnomalyDetectorService:
     Region: !Ref primaryRegion
   Parameters:
     Subscriber: !GetAtt CurrentAccount.RootEmail
+
+CostAndUsageReports:
+  Type: update-stacks
+  Template: cur.yaml
+  StackName: !Sub '${resourcePrefix}-CUR-Monthly'
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref primaryRegion
```

Comment:
we can probably remove this everything deploys to `primaryRegion` by default.  you only need to  set this if you want to deploy to non us-east-1 regions